src/main/java/Deadline.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+
+    private LocalDateTime by;
+    public Deadline(String name, LocalDateTime by) {
+        this(name, by, false);
+    }
+
+    public Deadline(String name, LocalDateTime by, boolean mark) {
```

Comment:
Perhaps a more boolean sounding variable name here? I noticed the same issue in several other places too.